examples/example_plugin/extended-veiwer.js

```diff
@@ -0,0 +1,48 @@
+import $ from 'jquery'
+
+export function ExtendedVeiwer(iv) {
```

Comment:
I've been migrating the main iXBRL viewer code base to use ES6 class syntax.  I think it would be nice to use that here.

Reply:
Awesome! The SEC representatives had mentioned that we should do this, I will look at what that entails for this plugin